src/style.css

```
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
```

Pay attention how lo load fonts using `@font-face`
```suggestion
@font-face {
  font-family: "Avenir";
  font-weight: 300;
  font-style: normal;
  src: url("./fonts/Avenir-Book.ttf") format("truetype");
}

@font-face {
  font-family: "Avenir";
  font-weight: 600;
  font-style: normal;
  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
}

body {
  font-family: "Avenir", sans-serif;
  font-weight: 300;
  font-size: 16px;
}
```